linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_ocredit/rule.yml

```diff
@@ -56,15 +56,21 @@ references:
     stigid@ubuntu2004: UBTU-20-010055
     vmmsrg: SRG-OS-000266-VMM-000940
 
-ocil_clause: 'ocredit is not found or not equal to or less than the required value'
+ocil_clause: 'ocredit is not found or is not equal to or less than the required value'
 
 ocil: |-
     To check how many special characters are required in a password, run the following command:
     <pre>$ grep ocredit /etc/security/pwquality.conf</pre>
-    The <tt>ocredit</tt> parameter (as a negative number) will indicate how many special characters are required.
+    The <tt>ocredit</tt> parameter (as a negative number) will indicate how many special
+    characters are required.
     The DoD and FISMA require at least one special character in a password.
     This would appear as <tt>ocredit = -1</tt>.
 
+fixtext: |-
+    Add or modify the "ocredit" option line in /etc/security/pwquality.conf to have the required
+    value, like in the following example:
+    ocredit = -1
```

Comment:
```suggestion
    ocredit = {{{ xccdf_value("var_password_pam_ocredit") }}}
```

Reply:
@ggbecker I tried this but in this way it is not translated to the spreadsheet by the `create_srg_export.py`. Was it expected to be translated? If so, I can check the script.

Reply:
Currently, the default values don't work. 

Related issues #8530 and #8612

Reply:
Thanks for mentioning these issues @Mab879. So, I can set the variable with the actual value (-1) since, according to the #8612, the variable option is not rendering. Is that ok for now? If so, we may only have to adjust again when the #8612 is fixed.

Reply:
In my opinion is better to set the correct value here already. We are going to fix these issues at some point and select the any variable that is required.

Reply:
I agree with @ggbecker. I have a fix in the works. See #8635

Reply:
Agreed. So, I will update the PR.